frontend/src/v5/ui/routes/viewer/toolbar/toolbar.styles.ts

```diff
@@ -65,3 +69,33 @@ export const ExpansionContainer = styled.div`
 		border-left: solid 1px ${({ theme }) => theme.palette.base.light};
 	}
 `;
+
+export const ClearIcon = styled(PlusIcon)`
+	transform: rotate(45deg);
```

Comment:
Any reason you're rotating the Plus Icon instead of using close-outlined?

Reply:
Simply because the `PlusIcon` is a viewer icon, so I know I am using the same styles